### DISCUSSION TITLE
Error is thrown for ZIP Archive created on MacBook

### DISCUSSION THREAD

**Discussion by arsicaleksa** [pimcore/pimcore#9162] [Q&A / Support]
# Bug Report

IMPORTANT: DO NOT SUBMIT ISSUES WHICH WERE NOT VERIFIED ON THE LATEST VERSION OF PIMCORE! 
Bug reports that do not meet the conditions listed below will be closed/deleted without comment.

- [x] Do not submit reports for unsupported versions, supported versions can be viewed here https://github.com/pimcore/pimcore/milestones
- [x] This is not a security issue / vulnerability -> use [this form](https://pimcorehq.wufoo.com/forms/pimcore-security-report/) instead
- [x] Behavior is reproducible on https://demo.pimcore.fun/admin/ (admin/demo)
or on a clean install of the latest build (out of your project, to avoid problems with custom code, plugins, ...)
- [x] You're not able to fix the problem yourself and send us a pull request instead of filing an issue.
- [x] There's no existing ticket for the same issue
- [x] Remove unused sections from below and this bullet points, submit only clean reports

### Expected behavior
When I create ZIP archive on MacBook with images I should be able to use `Upload ZIP Archive` feature for uploading assets from ZIP.
### Actual behavior
Error is thrown when I attempt to upload assets from ZIP archive with `Upload ZIP Archive` feature.
`The "/var/www/html/var/tmp/zip-import/MyImages/" file does not exist or is not readable.`
### Steps to reproduce
Create ZIP archive on MacBook
Use `Upload ZIP Archive` feature for uploading assets from ZIP

**Comment by dvesh3** [comment]
@arsicaleksa seems missing file permissions https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Permissions.html#page_File-Permissions

and if you are dealing with heavy files, then please check php settings `upload_max_filesize` & `post_max_size`
